includes/rest-api/class-sensei-rest-api-setup-wizard-controller.php

```diff
@@ -195,6 +195,7 @@ public function get_data() {
 			],
 			'features'        => [
 				'selected' => $user_data['features'],
+				'options'  => $this->setup_wizard->get_sensei_extensions(),
```

Comment:
The new `options` should be added to the `features` object in `get_schema` (which also has a old `plugins` property now that can be removed).

Reply:
Good catch! Fixed here: https://github.com/Automattic/sensei/pull/3176/commits/70a52d75ae492eeb0e5b90af315dee8a88f04412